maven-plugin/src/main/java/org/openmrs/maven/plugins/Setup.java

```diff
@@ -417,26 +371,112 @@ private void setDebugPort(Server server) {
         }
     }
 
-    private DistroProperties extractDistroToServer(Server server, boolean isCreatePlatform, File serverPath) throws MojoExecutionException, MojoFailureException {
-        DistroProperties distroProperties;
-        if(DistroHelper.isRefapp2_3_1orLower(server.getDistroArtifactId(), server.getVersion())){
-            distroProperties = new DistroProperties(server.getVersion());
-        } else {
-            distroProperties = distroHelper.downloadDistroProperties(serverPath, server);
+    private void setServerVersionsFromDistroProperties(Server server, DistroProperties distroProperties) throws MojoExecutionException {
```

Comment:
This is just a renaming of the function `configureVersion` (nothing to do with `extractDistroToServer`, which has been integrated into `resolveDistroProperties`).